index.html

```diff
@@ -7,61 +7,73 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script src="https://kit.fontawesome.com/4e655bdddb.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="./style.css">
-    <title>Portfolio-Setup </title>
+    <title>Portfolio-Setup</title>
 </head>
 
 <body>
     <!--header section starts-->
     <header>
         <h2 class="logo"><a href="#">SP</a></h2>
         <nav>
-            <div class="hamberger">
-            <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <svg id="show-nav-btn" class="nav-menu-button" width="24" height="24" viewBox="0 0 24 24" fill="none"
+                xmlns="http://www.w3.org/2000/svg">
                 <path fill-rule="evenodd" clip-rule="evenodd"
                     d="M3 7C3 6.44772 3.44772 6 4 6H20C20.5523 6 21 6.44772 21 7C21 7.55228 20.5523 8 20 8H4C3.44772 8 3 7.55228 3 7ZM3 12C3 11.4477 3.44772 11 4 11L20 11C20.5523 11 21 11.4477 21 12C21 12.5523 20.5523 13 20 13L4 13C3.44772 13 3 12.5523 3 12ZM4 16C3.44772 16 3 16.4477 3 17C3 17.5523 3.44772 18 4 18H20C20.5523 18 21 17.5523 21 17C21 16.4477 20.5523 16 20 16H4Z"
                     fill="#6070FF" />
             </svg>
-        </div>
+            <!-- Mobile Navbar Starts Here-->
+            <div id="mobile-menu" class="mobile-menu--hide">
+                <i id="hide-nav-btn" class="fa-solid fa-x mobile-menu-close-icon"></i>
+                <ul id="mobile-menu-links">
+                    <li><a href="#my-portfolio">Portfolio</a></li>
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#contact">Contact</a></li>
```

Comment:
- [ ] Great job with the design. :+1: These two links are not navigating to the corresponding sections on the page. I think this is happening because you do not have the `id` attributes on the sections. Kindly add the id attributes to the sections so the links can navigate to the sections as suggested in the project requirements. ✍️

Reply:
I added id attributes to the nav link section.